File: Python Programming/problems on numbericals/problem19.py
'''

19. Write a program which accept width & height of rectangle from user and calculate its area. 
	(Area = Width * Height) 

	Author : Milind Nalawade
	
	Date:26/06/2021
	
	Language : Python 3.9.0
	
'''
#======
#Logic
#======

def Area(w,h):
	return w*h
#=============
#Entry point function
#=============

def main():
	w=int(input("Enter width : "))
	h=int(input("Enter height : "))
	ret=Area(w,h)
	print("Area of the rectangle is : ",ret)
	
#======
#starter
#======
if __name__=="__main__":
	main()

File: C Programming/problem on numbericals/problem19.c
/*

19. Write a program which accept width & height of rectangle from user and calculate its area. 
	(Area = Width * Height) 

	Author : Milind Nalawade
	
	Date:26/06/2021
	
	Language : C programming
	
*/	

#include<stdio.h>

//Logic
float Area(float w,float h)
{
	return (w*h);
}

// Entry point function

int main()
{
	float w=0.0,h=0.0,ret=0.0;
	printf("Enter width : ");
	scanf("%f",&w);
	printf("Enter heigth : ");
	scanf("%f",&h);
	ret=Area(w,h);
	printf("Area of rectangle is : %.3f",ret);
	return 0;
}